kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -269,9 +269,7 @@ public class KafkaRestConfig extends RestConfig {
           + " Use 0 for no timeout";
   public static final String SIMPLE_CONSUMER_POOL_TIMEOUT_MS_DEFAULT = "1000";
 
-  // TODO: change this to "http://0.0.0.0:8082" when PORT_CONFIG is deleted.
```

Comment:
What implications is setting the listener default to a value, rather than null, going to have on users?

Reply:
There should be no effect other than `port` is completely ignored now. Before it was empty to give users to opportunity to use `port` (by setting `listeners` to empty).